new_devsurvey/src/components/HeaderComponent.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+	<div>
+		<div class="d-flex justify-content-center">
+			<div class="container-fluid">
+				<div class="d-flex justify-content-center mt-5 mb-4">
+					<h1 class="ms-3" style="font-family: 'Poppins', sans-serif; font-weight: 600; color: #2F80ED">
+						Dev survey
+					</h1>
+				</div>
+
+				<div class="d-flex justify-content-center mb-5">
+					<a href="https://opencollective.com/osscameroon/donate" target="_blank">
+						<img src="https://opencollective.com/webpack/donate/button@2x.png?color=white" width=200 />
+					</a>
+				</div>
+
+				<div class="row d-flex justify-content-center text-secondary">
+					<p style="text-align: center">You are a cameroonian developer, please take this 3 minutes survey <a
```

Comment:
We just need to increase the police

Reply:
The best thing would be to finish a component before pushing, but here, it seems that the diagram has not been completed